### ISSUE TITLE
Mailchimp Successful Message Fails Color Contrast

### ISSUE THREAD

**Issue by marcusrelacion** [AIGAdc/dcdesignweek.org#298]
**Describe the bug**
White text on a green background doesn't have an acceptable color contrast

**To Reproduce**

When subscribing to the newsletter and updates, upon successful submission, the message that appears as white text on a green background.

**Expected behavior**
Text to be legible

**Screenshots**

![image](https://user-images.githubusercontent.com/6185899/88331495-0f134480-ccfb-11ea-960d-d659d8ce14ea.png)

**Comment by marcusrelacion** [comment]
This was fixed in #297
